underscore.js

```diff
@@ -1370,6 +1370,24 @@
 
   _.property = property;
 
+  // Generates a function for a given object that returns the passed function's return value.
+  // Accepts a string or function literal for value.
+  // If value is a string, function assumes (and verifies) this string is a method name on the
+  // referenced object.
+  _.method = function(value) {
+    var isFunction = this.isFunction,
+        rest = this.rest,
+        func;
+
+    return function(obj) {
+      if (obj == null) { return; }
```

Comment:
This method responds to both a string literal (a method name) or a function literal.   This check is to sort out if the value passed is a function literal, so that later we know what to call apply on.